frontend/libs/studio-components/src/components/StudioCodelistEditor/StudioCodeListEditor.test.tsx

```diff
@@ -254,6 +256,22 @@ describe('StudioCodeListEditor', () => {
     await user.type(validValueInput, 'new value');
     expect(onChange).not.toHaveBeenCalled();
   });
+
+  it('Does trigger onInvalid if the code list is invalid', async () => {
+    const user = userEvent.setup();
+    renderCodeListEditor({ codeList: codeListWithDuplicatedValues });
+    const validValueInput = screen.getByRole('textbox', { name: texts.itemValue(3) });
+    await user.type(validValueInput, 'new value');
+    expect(onInvalid).toHaveBeenCalled();
```

Comment:
Her bør vi også sjekke at funksjonen blir kalt riktig antall ganger, for å verifisere at den blir utløst på riktig vis. Jeg ville ventet at den ble utløst ved hver endring, altså like mange ganger som lengden på den nye verdien.
```suggestion
    const newValue = 'new value';
    await user.type(validValueInput, newValue);
    expect(onInvalid).toHaveBeenCalledTimes(newValue.length);
```

Reply:
Lurt! Takk for tips ⭐